src/postgraphile/postgraphile.ts

```diff
@@ -45,6 +46,13 @@ type PostGraphileOptions = {
   // `json` (which causes the stack to become an array with elements for each
   // line of the stack).
   showErrorStack?: boolean,
+  // Enables ability to modify errors before sending them down to the client
+  // optionally can send down custom responses
```

Comment:
Please add `/* @middlewareOnly */` as it only affects the HTTP middleware (not the createPostGraphileSchema function)